core-rust/mesh-api-server/src/mesh_api/handlers/mempool_transaction.rs

```diff
@@ -28,10 +28,10 @@ pub(crate) async fn handle_mempool_transaction(
             )),
         );
     } else {
-        payload_hashes.get(0).unwrap()
+        payload_hashes.first().unwrap()
```

Comment:
It looks like you must have run `clippy` at a different (more recent?) version compared to the code base.

This isn't a problem, but in future, can we create a separate PR for reformatting changes, if we want to do them? This avoids too much noise in the PR :)